.gitignore

```diff
@@ -0,0 +1,9 @@
+*.рус
+*.pyc
+*.idea'.idea'
```

Comment:
You have commited .idea files again

Reply:
Working on that problem, but how did I do that if .idea added to .gitignore?

Reply:
You also have `.gitignore` in the `.idea` folder and local `.gitignore` overwrites `.gitignore` in the parent folder. The easies way to avoid this is to not use `git commit -a -m` or `git add --all` completely